doc/version_history.rst

```diff
@@ -2,6 +2,12 @@
 Version History
 ===============
 
+v0.5.6
```

Comment:
I think this is a quite substantial change. Consider at least increasing the minor revision, `v0.6.0`

Reply:
Done in the commit: [2ce50af](https://github.com/lsst-ts/ts_m2/pull/28/commits/2ce50af7b174ca88c63068bb94e185ac8586ddb9). Thanks!